renovate.json

```diff
@@ -1,82 +1,24 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "semanticCommits": "enabled",
-  "semanticCommitType": "build",
-  "semanticCommitScope": "",
-  "dependencyDashboard": true,
-  "automerge": false,
-  "separateMajorMinor": false,
-  "timezone": "America/Tijuana",
-  "labels": ["comp: build", "renovate managed", "action: merge"],
-  "keepUpdatedLabel": "renovate managed",
-  "rebaseWhen": "conflicted",
-  "lockFileMaintenance": {
-    "enabled": true
-  },
-  "ignorePaths": ["bazel/integration/tests/**"],
-  "ignoreDeps": ["rules_pkg", "@angular-devkit/build-angular"],
-  "baseBranches": ["main"],
+  "extends": ["github>angular/dev-infra//renovate-preset/default.json5"],
+
   "postUpgradeTasks": {
     "commands": [
-      "git restore bazel/pnpm-lock.yaml",
+      "git restore bazel/pnpm-lock.yaml .npmrc",
       "yarn install",
       "yarn bazel sync --only=repo || true",
       "yarn update-generated-files"
     ],
-    "executionMode": "update"
+    "executionMode": "branch"
```

Comment:
We do not want the post upgrade to run for each dependency, but instead we want it to run for each branch. See: https://docs.renovatebot.com/configuration-options/#executionmode